README.markdown

```diff
@@ -54,14 +54,25 @@ Usage
       }
     });
 
+    // supports multiple roles in query
+    acl.query(["member", "admin"], "blog", "create", function(err, allowed) {
+        if (allowed) {
+          // creating allowed!
+        } else {
+          // no creating allowed!
+        }
+    });
+
 Role and Resource Discovery
 ---------------------------
 
 If you are more of an object-oriented programmer and prefer to use objects
 to represent your roles and resources, then you're in luck! Virgen-ACL can
 discover roles and resources from your objects so long as your role objects
-contain the property `role_id` OR a function `getRoleId()` and your resource
+contain the property `role_id` OR a function `getRoleId()`  and your resource
 objects contain the property `resource_id` OR a function `getResourceId()`.
+Valid value types for role_ids are strings, `null` or arrays of strings. Valid
```

Comment:
Is the use of plurals correct here? `strings` and `arrays` makes it sound like multiple arguments are allowed, but that wouldn't make sense in this case.